tests/puppeteer/pages/FO/FO_cart.js

```diff
@@ -0,0 +1,35 @@
+const CommonPage = require('../commonPage');
```

Comment:
why `FO/FO_` it's redundant.
Rename file please.

Reply:
These modifications will be done in a separate PR.